docs/releases/1.10.1.txt

```diff
@@ -56,3 +56,5 @@ Bugfixes
 * Reallowed the ``{% for %}`` tag to unpack any iterable (:ticket:`27058`).
 
 * Fixed ``makemigrations`` crash if a database is read-only (:ticket:`27054`).
+
+* Fixed duplicated managers in `Model._meta.managers` (:ticket:`27073`).
```

Comment:
Issue is, `_meta.manager` is a low-level implementation detail (duplicates was actually something I purposely didn't fight against), your phrasing gives me the impression there are duplicates in the public facing APIs.